base/src/main/java/uk/ac/ebi/atlas/search/SemanticQueryTerm.java

```diff
@@ -23,7 +23,7 @@ public static SemanticQueryTerm create(String value, String category) {
     public String asGxaIndexQueryLiteral() {
         return hasNoCategory()
                 ? String.format("property_value:\"%s\"", value())
-                : String.format("property_name:\"%s\" AND property_value:\"%s\"", category(), value().replace(":", "\\:").replace("[", "\\[").replace("]", "\\]"));
+                : String.format("property_name:\"%s\" AND property_value:\"%s\"", category(), value());
```

Comment:
I saw this on my phone and was concerned, I looked at it again and I'm not concerned about the change you did but I think we should escape the " that can potentially be inside value():
```
grep \" /ebi/ftp/pub/databases/microarray/data/atlas/bioentity_properties/ensembl/*.ensgene.tsv
``` 
returns some property values, and we don't tidy these values up on ingestion.

Reply:
I couldn’t find any match in `analytics` with `identifier_search:*\"*` or `conditions_search:*\"*`, and none in `bioentities` with `property_value:*\"*`, but I think this will silence some of the nightly exceptions we get from bad syntax searches.